holoviews/plotting/bokeh/element.py

```diff
@@ -572,7 +581,7 @@ def _set_active_tools(self, plot):
                 plot.toolbar.active_scroll = tool
             if isinstance(tool, tools.Tap):
                 plot.toolbar.active_tap = tool
-            if isinstance(tool, tools.Inspection):
+            if not bokeh3 and isinstance(tool, tools.Inspection):
```

Comment:
Not supported in Bokeh 3?

Reply:
Correct. `from bokeh.models.tools import Inspection` gives an `ImportError` in Bokeh 3. 

But looking at it, it could be renamed `InspectTool`.  I can't find clear information in the [migration guide](https://github.com/bokeh/bokeh/wiki/Migration-Guides). 

@mattpap can you tell me if this is the case?

Reply:
There was previously:
```py
# TODO: deprecated, remove at bokeh 3.0
Inspection = InspectTool
```
though probably wasn't deprecated properly.

Reply:
Can see `from bokeh.models.tools import InspectTool` also works in Bokeh 2.4.0 (lower pin of panel=0.13.1), so I will update it to that.